openpdf/src/main/java/com/lowagie/text/Document.java

```diff
@@ -730,145 +695,152 @@ public boolean addModificationDate() {
     /**
      * Adds the current date and time to a Document.
      *
-     * @return <CODE>true</CODE> if successful, <CODE>false</CODE> otherwise
+     * @return  <CODE>true</CODE> if successful, <CODE>false</CODE> otherwise
      */
     public boolean addModificationDate(PdfDate date) {
         try {
             return add(new Meta(Element.MODIFICATIONDATE, date.toString()));
-        } catch (DocumentException de) {
+        }
```

Comment:
Please see the coding-style here, as changing the style makes reviewing your PR a lot more difficult:

- https://github.com/LibrePDF/OpenPDF/tree/master/config
- https://google.github.io/styleguide/javaguide.html
- https://github.com/LibrePDF/OpenPDF/blob/master/CONTRIBUTING.md